ui/src/app/views/concept-set-details/component.spec.ts

```diff
@@ -3,12 +3,13 @@ import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing'
 import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 import {By} from '@angular/platform-browser';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
-import {ActivatedRoute, Router} from '@angular/router';
+import {ActivatedRoute} from '@angular/router';
```

Comment:
Can we also pull activated route? Or is that needed for the factory? Are you planning to handle that in a follow up?

Reply:
There's still one piece of data being pulled from the route in this component, so this has to stay for now.